myosuite/envs/myo/base_v0.py

```diff
@@ -70,7 +70,7 @@ def step(self, a):
         muscle_a = a.copy()
 
         # Explicitely project normalized space (-1,1) to actuator space (0,1) if muscles
-        if self.sim.model.na:
+        if self.sim.model.na and self.normalize_act:
```

Comment:
Did we check that this was keeping other envs the same?

Reply:
I have made a script to test all the environments. List of envs to test are taken from test_myo.py I have found that some environments are affected by this change and I will summarize them here:

1. motorFingerReachFixed-v0 - Not normalizing muscles when normalize_act = True
2. motorFingerReachRandom-v0 - Not normalizing muscles when normalize_act = True
3. motorFingerPoseFixed-v0 - Not normalizing muscles when normalize_act = True
4. motorFingerPoseRandom-v0 - Not normalizing muscles when normalize_act = True
5. myoElbowPose1D6MExoRandom-v0 - Outputs are zeros despite being normalized
6. myoSarcElbowPose1D6MExoFixed-v0 - Outputs are zeros despite being normalized
7. myoFatiElbowPose1D6MExoFixed-v0 - Outputs are zeros despite being normalized
8. myoSarcElbowPose1D6MExoRandom-v0 - Outputs are zeros despite being normalized
9. myoFatiElbowPose1D6MExoRandom-v0 - Outputs are zeros despite being normalized

[EnvTesting.pdf](https://github.com/facebookresearch/myosuite/files/11513495/EnvTesting.pdf)

Reply:
More testing showed no problems with the environments. The zero values are due to the actuators being motors (which do not require normalization).

[EnvTesting_FInalCheck.pdf](https://github.com/facebookresearch/myosuite/files/11518661/EnvTesting_FInalCheck.pdf)